CONTRIBUTING.md

```diff
@@ -43,6 +49,15 @@ The process for contributing code changes works as follows:
 4. After receiving maintainer approval, the PR is merged after a short grace
    period.
 
+You may want to see some details on the [Branching Strategy](./readme/Releases.md#branching-strategy).
+
+When developing new features, create an issue or comment on an existing issue to let others know what you are doing.
+
+During development, create a fork and use the master as base. Create a pull request for each fix.
+
+_Do not_ create big pull requests. As long as you don't break anything features also can be
+merged, even if they are not 100% finished.
```

Comment:
These lines seem randomly placed and should either be entirely omitted or incorporated into their respective sections.

Reply:
Ah good point - they were copied from a different section but i don't think they need to be here :+1: